src/room/useLoadGroupCall.ts

```diff
@@ -53,7 +54,7 @@ export const useLoadGroupCall = (
       const waitPromise = new Promise<Room>((resolve) => {
         const onRoomEvent = async (room: Room) => {
           if (room.roomId === roomId) {
-            client.removeListener(ClientEvent.Room, onRoomEvent);
+            client.removeListener(GroupCallEventHandlerEvent.Room, onRoomEvent);
```

Comment:
Maybe this is just GitHub on Android lying about what lines were changed, but how does us removing a different event listener fix this?

Reply:
GitHub on Android is not lying, I just failed to update the right line, and also failed to actually commit the js-sdk change anyway...

Reply:
Ah, I see